project/AppDependencies.scala

```diff
@@ -5,7 +5,7 @@ object AppDependencies {
   import play.core.PlayVersion
 
   private val silencerVersion      = "1.7.9"
-  private val bootstrapPlayVersion = "7.2.0"
+  private val bootstrapPlayVersion = "7.3.0"
   private val hmrcMongoPlayVersion = "0.71.0"
```

Comment:
Can be -> uk.gov.hmrc.mongo:hmrc-mongo-play-28           : 0.71.0  -> 0.72.0

Reply:
Yh just seen that there is a new release. It was 0.71.0 when i checked. Will bump :)